tool/conda/singa/conda_build_config.yaml

```diff
@@ -27,12 +27,16 @@ CONDA_BUILD_SYSROOT:
 cudnn:                      # [linux]
     - "7.3.1 cuda10.0_0"    # [environ.get("CUDA")=="10.0"]
     - "7.3.1 cuda9.0_0"     # [environ.get("CUDA")=="9.0"]
-mkldnn:
-    - 0.14
 python:
     - 3.6
     - 3.7
+nccl:
+    - 2.4.8.1
+mpich:
+    - 3.3.2
 build_str:
-    - "cudnn7.3.1_cuda10.0"   # [environ.get("CUDA")=="10.0"]
-    - "cudnn7.3.1_cuda9.0"    # [environ.get("CUDA")=="9.0"]
-    - "cpu"              # [environ.get("CUDA", "")== ""]
\ No newline at end of file
+    - "cudnn7.3.1_cuda10.0"   # [environ.get("CUDA")=="10.0"] && [environ.get("DIST")=="OFF"]
+    - "cudnn7.3.1_cuda9.0"    # [environ.get("CUDA")=="9.0"] && [environ.get("DIST")=="OFF"]
+    - "cpu"              # [environ.get("CUDA", "")== ""]
+    - "cudnn7.3.1_cuda10.0_nccl2.4.8.1_mpich3.3.2"     # [environ.get("CUDA")=="10.0"] && [environ.get("DIST")=="ON"]
```

Comment:
did you do the test on gpu machines?
I mean compile singa dist version using conda on local gpu machine.

Reply:
Yes, did the build test in the docker container for conda-build, on 1.cpu version build, 2.gpu version build, and 3.gpu+nccl version build